.gitignore

```diff
@@ -29,4 +29,7 @@ nuget.exe
 *.ncrunchsolution
 *.*sdf
 *.ipch
-*.sln.ide
\ No newline at end of file
+.settings
+*.sln.ide
+node_modules
+**/[Cc]ompiler/[Rr]esources/**/*.js
```

Comment:
Because the JS file in source is unminified (in the js folder). Grunt then minifies the file to the compiler resources folder.